VirtuelleMessstelle/module.php

```diff
@@ -139,12 +139,20 @@ public function GetConfigurationForm()
     {
         //Add options to form
         $jsonForm = json_decode(file_get_contents(__DIR__ . '/form.json'), true);
-        $options = $this->GetOptions();
-        $jsonForm['elements'][0]['options'] = $options;
-        $jsonForm['elements'][1]['columns'][1]['edit']['options'] = $options;
-        if (count($options) > 0) {
-            $jsonForm['elements'][1]['columns'][1]['add'] = $options[0]['value'];
+
+        //If the module "SyncMySQL" is install, get other options
+        if (IPS_ModuleExists('{7E122824-E4D6-4FF8-8AA1-2B7BB36D5EC9}')) {
```

Comment:
Ich fände es optisch schöner, wenn wir die 700px in der Breite standardmäßig weglassen und nur in dem "if" setzen. Das passt dann besser zu unseren anderen Modulen.